failed to install

Error log below:

```
$ jupyter labextension install jupyterlab-mp4
Node v8.11.3

> /Users/jamesyang/.nvm/versions/node/v8.11.3/bin/npm pack jupyterlab-mp4
npm ERR! code E404
npm ERR! 404 Not Found: jupyterlab-mp4@latest

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jamesyang/.npm/_logs/2018-11-24T14_23_39_061Z-debug.log

Errored, use --debug for full output:
ValueError: "jupyterlab-mp4" is not a valid npm package
```

--- a/README.md
+++ b/README.md
@@ -9,7 +9,7 @@ A JupyterLab extension for rendering mp4 files.
 ## Installation
 
 ```bash
-jupyter labextension install jupyterlab-mp4
+jupyter labextension install @jupyterlab/mp4-extension
 ```
 
 ## Development

